apps/webview/src/utils/string.ts

```
@@ -0,0 +1,41 @@
+};
+
+// camel to kebab
+export const convertCamelToKebab = (str: string) => {
+  return convertCamelToSnake(str).replaceAll('_', '-');
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const convertCamelObjToSnake = (obj: Record<string, any>) => {
+  return Object.entries(obj).reduce((acc, [key, value]) => {
+    const convertedKey = convertCamelToSnake(key);
+    return { ...acc, [convertedKey]: value };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)